example/src/main/java/org/xmtp/android/example/connect/ConnectWalletViewModel.kt

```diff
@@ -31,10 +68,25 @@ class ConnectWalletViewModel : ViewModel() {
         }
     }
 
+    @UiThread
+    fun connectWallet() {
+        viewModelScope.launch(Dispatchers.IO) {
+            _uiState.value = ConnectUiState.Loading
+            try {
+                val wallet = WCAccount(walletConnectKit)
+                val client = Client().create(wallet)
```

Comment:
Unfortunately this is failing for me rn but wanted to push something up to get some eyes on it while I debug!

Reply:
How is it failing? Because in the `PrivateKey` implementation of `SigningKey` you are using `eth_sign` afaik, and walletconnect only supports `personal_sign`.

Reply:
Was able to resolve the issue! It was a null pointer with the signature creation.